core/src/test/java/com/linecorp/armeria/client/HttpClientIntegrationTest.java

```diff
@@ -737,20 +709,20 @@ void testUpgradeRequestExecutesLogicOnlyOnce() throws Exception {
         final AggregatedHttpResponse response = client.execute(
                 AggregatedHttpRequest.of(HttpMethod.GET, "/only-once/request")).aggregate().get();
 
-        assertEquals(HttpStatus.OK, response.status());
+        assertThat(response.status()).isEqualTo(HttpStatus.OK);
 
         clientFactory.close();
     }
 
     @Nested
     @TestInstance(Lifecycle.PER_CLASS)
-    class JettyInterop {
+    static class JettyInterop {
 
-        org.eclipse.jetty.server.Server jetty;
+        Server jetty;
```

Comment:
I had figured someone down the line will get confused when trying to initialize an Armeria `Server` so didn't import.

Reply:
Ah, OK. Let me revert this.

Reply:
Thanks - apparently not importing was also confusing so let's add a one-line comment too :)

Reply:
#2097